.github/workflows/CI.yml

```diff
@@ -1,42 +1,47 @@
 name: CI
 on:
+  pull_request:
+    branches:
+      - main
+    paths-ignore:
+      - 'docs/**'
   push:
     branches:
       - main
-    tags: ['*']
-  pull_request:
-concurrency:
-  # Skip intermediate builds: always.
-  # Cancel intermediate builds: only if it is a pull request build.
-  group: ${{ github.workflow }}-${{ github.ref }}
-  cancel-in-progress: ${{ startsWith(github.ref, 'refs/pull/') }}
+    paths-ignore:
+      - 'docs/**'
+  schedule:
+    - cron: '53 20 * * 5'
 jobs:
   test:
-    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }} - ${{ github.event_name }}
     runs-on: ${{ matrix.os }}
+    env:
+      GROUP: ${{ matrix.group }}
     strategy:
       fail-fast: false
       matrix:
+        group:
+          - Core
         version:
-          - '1.6'
```

Comment:
ADTypes is compatible with 1.6 so it needs to be tested on 1.6. There are packages outside of SciML that depend on it and want to support 1.6 (first and foremost DifferentiationInterface). I don't approve this change